### DISCUSSION TITLE
Logic negation

### DISCUSSION THREAD

**Discussion by nandi95** [fullstackfool/vacl#11] [Ideas]
- **I'm submitting a ...**
  [ ] bug report
  [x] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**
Add option for modifiers that negate the evaluation of roles and permissions
`v-can.not="edit-post"`

**Comment by fullstackfool** [comment]
[`v-cannot`](https://github.com/fullstackfool/vacl#directives)?

**Comment by nandi95** [comment]
Indeed, but it's a nice addition and allows people to write in this style if they prefer.

**Comment by fullstackfool** [comment]
Maybe. Plan for the repo is to be simplistic and maybe a little opinionated in that simplicity. Will have a look/think 👍
